x-pack/plugins/infra/public/pages/metrics/layouts/nginx.ts

```diff
@@ -73,7 +84,9 @@ export const nginxLayoutCreator: InfraMetricLayoutCreator = theme => [
             reqPerConns: {
               color: theme.eui.euiColorVis1,
               type: InfraMetricLayoutVisualizationType.bar,
-              name: 'reqs per conn',
+              name: i18n.translate('xpack.infra.nginxMetrics.visConfig.requestsAliasName', {
+                defaultMessage: 'reqs per conn',
```

Comment:
Could you please add a screenshot where this label is displayed?

Reply:
it seems that Nginx is not available for me, so i think it will seem like this
![image](https://user-images.githubusercontent.com/23155410/48409391-c4cb7900-e74c-11e8-87db-28cba40adbb1.png)